style.css

```diff
@@ -0,0 +1,35 @@
+#body{
+    background-image: url('https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg');
+    opacity: 0.8;
+    color: white;
+    }
+    #body-wrapper{
+        background-color:#1b1b32;
+        width: 50%;
+        margin: auto;
+    }
+    #main{
+        text-align: center;
+    }
+    #survey-form{
+        display: flex;
+        flex-direction:column;
+        max-width: 40%;
+        width: 100%;
+        margin: auto;
+    }
+    input{
+        height: 30px;
+    }
+    label{
+    margin-top: 10px;
+    }
```

Comment:
I would suggest you use class attributes and you can use selectors for multiple elements.

Reply:
@Santosh-Konappanavar :wave: 

- [ ] Kindly phrase the comments politely as suggestions instead of orders.
  - For example:
    - This is a polite suggestion: "I think it would be better to use semantic tags".
 
- [ ] Kindly use reasoning `why` students need to consider your suggestion. Why do students need to use class attributes and selectors? Use reasoning here and you can add any referral link from where students can get hints.